source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRFactSetRenderer.mm

```diff
@@ -65,6 +65,12 @@ + (ACRUILabel *)buildLabel:(NSString *)text
             content = [[NSMutableAttributedString alloc] initWithData:htmlData options:options documentAttributes:nil error:nil];
             // Drop newline char
             [content deleteCharactersInRange:NSMakeRange([content length] - 1, 1)];
+            [content enumerateAttribute:NSFontAttributeName
```

Comment:
it looks like this block of code is the same across all 3 files, is there some way to create a static function that does the same?

Reply:
okay